composer.json

```diff
@@ -2,21 +2,21 @@
     "name": "ua-parser/uap-php",
     "description": "A multi-language port of Browserscope's user agent parser.",
     "require": {
-        "php": ">=5.3.0",
+        "php": "^7.2",
         "composer/ca-bundle": "^1.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "<8",
-        "symfony/yaml": "^2.0 || ^3.0 || ^4.0",
-        "symfony/filesystem": "^2.0 || ^3.0 || ^4.0",
-        "symfony/finder": "^2.0 || ^3.0 || ^4.0",
-        "symfony/console": "^2.0 || ^3.0 || ^4.0"
+        "phpunit/phpunit": "^7 || ^8 || ^9",
+        "symfony/yaml": "^3.4 || ^4.3 || ^5.0",
```

Comment:
Is there a reason to not to drop symfony 3.x? Symfony 4.3+ requires PHP ^7.1.3, Somfony 5.0+ requires PHP 7.2.5

Reply:
Because Symfony 3.4 is a LTS, like the 4.4

Reply:
And 3.4 is the last version of the 3.x, but it can be dropped easily if we consider that this version will be deprecated on November

Reply:
But if we drop 3.x, we should force ^4.4 cause the 4.3 will be deprecated too in June ;)